_posts/2022-05-06-Setting-up-replication.md

```diff
@@ -0,0 +1,114 @@
+---
+layout: post
+title: How to set up replication from PostgreSQL to SAP HANA Cloud? 
+categories: [SAP BTP, SAP HANA Cloud, PostgreSQL]
+author_github: Kostya2702
+author: Konstantin Loshkarev
+---
+
+If you have a need to use new CAP technologies, and the database contains a lot of data, then you will need to replicate to SAP HANA Cloud. In this article, we will look at the complete setup using PostgreSQL without SSL as an example.
```

Comment:
If you need to use new CAP technologies and your database contains a lot of data then you have to replicate your date to SAP HANA Cloud. In this article we will look at the complete setup using PostgreSQL without SSL as an example.